python/tox.ini

```diff
@@ -0,0 +1,39 @@
+[tox]
+minversion = 2.0
+distshare = {homedir}/.tox/distshare
+# make sure to update environment list in travis.yml and appveyor.yml
+envlist =
+    py27
+    py35
+    lint
+
+[testenv]
+commands = pytest {posargs:tests}
+usedevelop = True
+deps = 
+    pytest
+    pdbpp
+
+[testenv:lint]
+skipsdist = True
+usedevelop = True
+basepython = python2.7
+deps =
+    flake8
+    # pygments required by rst-lint
+    pygments
+    restructuredtext_lint
+commands =
+    flake8 src/deltachat
+    flake8 tests/
+    # {envpython} scripts/check-rst.py
+
+[pytest]
+minversion = 2.0
```

Comment:
do you still run with 2.x?  I mean, is this actually tested?  pytest has been 3.x for a while now...